pyproject.toml

```diff
@@ -22,12 +22,12 @@ aiohttp = "^3.10.5"
 aiohttp-retry = "^2.8.3"
 tenacity = ">=8.4.1"
 cel-python = "^0.1.5"
-opentelemetry-api = "^1.27.0"
-opentelemetry-sdk = "^1.27.0"
-opentelemetry-instrumentation = "^0.48b0"
-opentelemetry-distro = "^0.48b0"
-opentelemetry-exporter-otlp = "^1.27.0"
-opentelemetry-exporter-otlp-proto-http = "^1.27.0"
+opentelemetry-api = "^1.28.0"
+opentelemetry-sdk = "^1.28.0"
+opentelemetry-instrumentation = ">=0.49b0"
+opentelemetry-distro = ">=0.49b0"
```

Comment:
this is the corresponding version for 28, but the key here is the `>=` instead of `^` to allow for users to use e.g. `0.50`